identities/models.py

```diff
@@ -183,7 +183,8 @@ def handle_optout(sender, instance, created, **kwargs):
         sender=None,
         event_name='optout.requested',
         payload={
-            'details': identity.details,
+            'identity_id': str(identity.id),
+            'identity_details': identity.details,
```

Comment:
the prefix makes more sense here given it's clarifying it's not the details of the optout